packages/dispatcher-converter/src/converter/amsDispatcherConfigConverter.js

```diff
@@ -1789,6 +1813,66 @@ class AEMDispatcherConfigConverter {
 
         return usedFiles;
     }
+    /**
+     * Scans the `available_vhosts` folder, and if a symbolic link doesn't already exist in the `enabled_vhosts` folder, one
+     * is created.
+     */
+    createVhostSymLinks(enabledVhostsPath, availableVhostsPath) {
+        let conversionStep = this.createVhostSymLinksSummaryGenerator();
+        let files = glob.sync(path.join(availableVhostsPath, "*.vhost")) || [];
+        files.forEach((file) => {
+            logger.info(
+                "Creating Vhost Symbolic Link in target folder for file : " +
+                    file
+            );
+            let fileName = path.basename(file);
+            this.FileOperationsUtility.createSymLink(
+                "../available_vhosts/" + fileName,
```

Comment:
Please reuse the constant `AVAILABLE_VHOSTS` instead of `available_vhosts`.

Also more importantly, we should not do any path concatenation (since paths separators are OS dependent). Always use `path.join()` for the same;.